### PR TITLE
Talha/pydantic model

### PR DESCRIPTION
**Pull request by mtalhatahir2001**
Ticket links:
https://app.asana.com/0/1204411152293714/1204439508142892
https://app.asana.com/0/1204411152293714/1204439508142886
https://app.asana.com/0/1204411152293714/1204439508142884

Created Pydantic models. 
User
Genre
Book

### CODE REVIEW

**Comment by mustansirMuz** [comment]
Change the base branch to dev...

File: src/schemas/user.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    Has attributes and validations as of DB.
+    """
+
+    id: Optional[int] = None
+    email: str = Field(min_length=0, max_length=32)
+    username: str = Field(min_length=0, max_length=32)
+    password: str = Field(min_length=0, max_length=8)
+    first_name: str = Field(min_length=0, max_length=32)
+    last_name: str = Field(min_length=0, max_length=32)
+    date_of_joining: datetime = Field()
```
Comment (mustansirMuz) [inline src/schemas/user.py:19]: Shouldn't date of joining be optional? If not passed then can be set to current date?

Comment (mustansirMuz) [reply src/schemas/user.py]: Yes but this also allows the user to provide an incorrect date of joining. We can have timezone support in datetime fields so time difference shouldn't be a problem imo

**Review by mustansirMuz — APPROVED** [review]
LGTM

**Review by SohaibkhanFolio3 — APPROVED** [review]
LGTM too

**Review by mustansirMuz — APPROVED** [review]
LGTM

**Pull request merged**